robottelo/cli/repository.py

```diff
@@ -60,7 +60,7 @@ def synchronize(cls, options, return_raw_response=None, timeout=3600000):
         cls.command_sub = 'synchronize'
         return cls.execute(
             cls._construct_command(options),
-            output_format='csv',
+            output_format='base',
```

Comment:
@vsedmik This has been fixed in https://github.com/SatelliteQE/robottelo/pull/13737 . You should not need this w/o anymore!